pkg/helm/charts/tests/centralservices/testdata/helmtest/central.test.yaml

```
@@ -85,3 +91,15 @@ tests:
     .securitycontextconstraints["stackrox-central"] | assertThat(. == null)
     .deployments["central"].spec.template.spec.affinity.nodeAffinity | .preferredDuringSchedulingIgnoredDuringExecution | assertThat(length == 5)
     verifyNodeAffinities(.deployments["central"])
+
+- name: Tenant ID should be set when env.managedServices is true
+  set:
+    env.managedServices: true
+  expect: |
+    envVars(.deployments.central; "central") | assertThat(has("ROX_TENANT_ID") == true)
```

Would this simpler thing work too?
```suggestion
    envVars(.deployments.central; "central") | assertThat(has("ROX_TENANT_ID"))
```

---

Yes, it would. It is there to rhyme with the `== false` test below.